routes/defRoute.js

```diff
@@ -1,14 +1,17 @@
 const express = require('express');
 const passport = require('passport');
-
-const { resServerError } = require('../shared/resServerError');
+const resServerError = require('../shared/resServerError');
```

Comment:
Why have you deleted this string?
const { resServerError } = require('../shared/resServerError'); 
This option is coorect.